config/initializers/monkeypatches/rspec_wait_handler.rb

```diff
@@ -0,0 +1,51 @@
+if defined?(RSpec::Wait::Handler) && defined?(Prosopite)
+  module RSpecWaitHandlerPatches
+    # We are monkeypatching this method:
+    # https://github.com/laserlemon/rspec-wait/blob/v1.0.1/lib/rspec/wait/handler.rb#L11-L31
+    # rubocop:disable Metrics
+    def handle_matcher(target, initial_matcher, message, &block)
+      start_time = Process.clock_gettime(Process::CLOCK_MONOTONIC)
+
+      answer = nil
+
+      Prosopite.pause do
+        matcher = RSpec.configuration.clone_wait_matcher ? initial_matcher.clone : initial_matcher
+
+        grandfather_method = method(:handle_matcher).super_method.super_method
```

Comment:
This is fairly ugly. I think there's probably a better way to do this. But I'm just going to go with this for now.

What's happening here is basically that, in the original method, `super` was called here. Since we are adding another entry to the inheritance chain with `prepend`, that `super` method (in the original code) is the "grandfather" method of this method.